sphinx_doc/setup_guide_centos.rst

```diff
@@ -151,6 +151,15 @@ Install PostgreSQL::
     #     restart : stop/start the database; used to read changes to core configuration files.
     #     reload  : reload pg_hba.conf file while keeping database running.
 
+    # login as superuser
+    su
+    
+    # Put the following code in the config file
+    vim /var/lib/pgsql/9.5/data/pg_hba.conf
+    local   all             all                               peer
+    host    all             all             127.0.0.1/32      ident
+    host    all             all             ::1/128           md5
+
```

Comment:
For this, I think we can move it into another page of documents, like troubleshooting. I will do that, though.